src/main/java/jalau/at18/azul/CenterTileBoard.java

```diff
@@ -0,0 +1,30 @@
+package jalau.at18.azul;
+import java.util.*;
+
+public class CenterTileBoard extends ArrayList<Character> {
```

Comment:
I've seen a `Tile` enum is being created, can this be `ArrayList<Tile>` instead?

Reply:
Yes, this will be changed when our pull requests are on the main, since we don't have the enum yet.

Reply:
Makes sense, thanks for clarification